lib/commands/messaging.js

```diff
@@ -2521,9 +3019,21 @@ messaging
     .option(`--scheduled-at <scheduled-at>`, `Scheduled delivery time for message in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) format. DateTime value must be in future.`)
     .action(actionRunner(messagingCreateSms))
 
+messaging
+    .command(`create-sms`)
+    .description(`Create a new SMS message.`)
+    .requiredOption(`--message-id <message-id>`, `Message ID. Choose a custom ID or generate a random ID with 'ID.unique()'. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can't start with a special char. Max length is 36 chars.`)
+    .requiredOption(`--content <content>`, `SMS Content.`)
+    .option(`--topics [topics...]`, `List of Topic IDs.`)
+    .option(`--users [users...]`, `List of User IDs.`)
+    .option(`--targets [targets...]`, `List of Targets IDs.`)
+    .option(`--draft [value]`, `Is message a draft`, (value) => value === undefined ? true : parseBool(value))
+    .option(`--scheduled-at <scheduled-at>`, `Scheduled delivery time for message in [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) format. DateTime value must be in future.`)
+    .action(actionRunner(messagingCreateSMS))
+
 messaging
     .command(`update-sms`)
-    .description(`Update an SMS message by its unique ID. This endpoint only works on messages that are in draft status. Messages that are already processing, sent, or failed cannot be updated. `)
+    .description(`[**DEPRECATED** - This command is deprecated. Please use 'messaging update-sms' instead] Update an SMS message by its unique ID. This endpoint only works on messages that are in draft status. Messages that are already processing, sent, or failed cannot be updated. `)
```

Comment:
Same here, probably due to `SMS` becoming uppercase, but in CLI it makes no difference